Shared/Samples/Download preplanned map area/DownloadPreplannedMapAreaView.swift

```diff
@@ -73,3 +73,12 @@ struct DownloadPreplannedMapAreaView: View {
             .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
     }
 }
+
+private extension Viewpoint {
```

Comment:
Probably keep this consistent with `private extension Envelope` in other samples.

Reply:
If the viewpoint is made a static method/property then `Viewpoint.targetGeometry` is not available. 
<img width="1282" alt="Screenshot 2023-09-06 at 1 38 36 PM" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/117859673/b2584577-6bac-43c8-be90-d5f30375e2c1">

Reply:
OK. Add a docstring to this method then.

Reply:
I added the docstring and renamed the method.